addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_primitive_attributes.py

```diff
@@ -192,6 +192,15 @@ def __gather_skins(blender_primitive, export_settings):
 
             # weights
             internal_weight = blender_primitive["attributes"][weight_id]
+            # normalize first 4 weights, when not exporting all influences
+            if not export_settings['gltf_all_vertex_influences']:
+                for idx in range(0, len(internal_weight), 4):
```

Comment:
This is normalizing weights 1-4, 5-8, 9-12, each group in batches of four, correct? That seems a little confusing, doesn't look like we should need this outer loop.

Reply:
Yep, it needs to normalize each bundle of 4. Not sure how you would do that without the outer loop in line 197

Reply:
Oh I see, `internal_weights` is the weights for all vertices, and each "bundle of 4" is one vertex? I think I'd misread and assumed this was normalizing WEIGHTS_0, then normalizing WEIGHTS_1, etc.